pantheon.yml

```diff
@@ -26,15 +26,42 @@ workflows:
   # The deployment event for Test and Live
   deploy:
     after:
+      - type: webphp
+        description: Rebuild cache before config import
+        script: private/scripts/drush_post_deploy_cache_rebuild.php
+      - type: webphp
+        description: Update database
+        script: private/scripts/drush_post_deploy_updatedb.php
       - type: webphp
         description: Import configuration from .yml files
-        script: private/scripts/drush_post_deploy.php
+        script: private/scripts/drush_post_deploy_config_import.php
```

Comment:
Maybe we should move a cache rebuild here, after this script too?

Reply:
Will add a CR here